build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -489,9 +492,13 @@ def from_json_object(json_object: Dict[str, Any]):
 
 @dataclass(frozen=True)
 class ModuleComponentSizes(object):
+  # File size in bytes.
   file_size: int
+  # VM component size in bytes.
   vm_component_size: int
+  # Const component size in bytes.
   const_component_size: int
+  # Total dispatch size in bytes.
   total_dispatch_component_size: int
```

Comment:
Can we put these in the field names rather than in comments? e.g. `file_bytes` `vm_component_bytes`, etc

Reply:
Done.